tcli/tcli_lib.py

```diff
@@ -1170,27 +1164,27 @@ def _Pipe(self, output, pipe=''):
 
     if not pipe:
       return output
-    else:
-      pipe = pipe.lstrip('|')
-      try:
-        p = subprocess.Popen(
-            [pipe], shell=True, close_fds=True,
-            stdin=subprocess.PIPE, stdout=subprocess.PIPE)
-      except IOError as error_message:
-        logging.error('IOerror opening pipe.')
-        if str(error_message):
-          self._PrintWarning(str(error_message))
-        return
 
-      try:
-        result, _ = p.communicate(bytes(output, 'utf-8'))
-        logging.debug('Output written to pipe:\n%s', output)
-        logging.debug('Output read from pipe:\n%s', result)
-        return result
+    pipe = pipe.lstrip('|')
```

Comment:
Nothing changed here - removing the redundant 'else' (line 1173) allowed this section to be out-indented.